install/0000_10_config-operator_01_imagepolicies-CustomNoUpgrade.crd.yaml

```diff
@@ -0,0 +1,402 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
These `install/` files are no longer needed, and should be dropped from this pull.  See [this thread pointing out that the script populating them was obsolete][1].

[1]: https://github.com/openshift/machine-config-operator/pull/4587#discussion_r1769161791